data/src/main/java/com/example/data/backendapi/controller/CustomerController.java

```diff
@@ -0,0 +1,22 @@
+package com.example.data.backendapi.controller;
+
+import com.example.data.backendapi.dto.Customer;
+import com.example.data.backendapi.service.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CustomerController {
+
+    @Autowired
```

Comment:
This is not recommended way to define injection. Prefer constructor injection instead.
One of the good reason for that it could simplify the unit testing. Also that will help you to keep the class under control and do not bloat it with many dependencies violating Low Coupling, Single Responsibility, Interface segregation principles.